core/block/import/markdown/anymark/markdown.go

```diff
@@ -233,6 +233,15 @@ func getCustomHTMLRules() []html2md.Rule {
 		simpleText, blockquote, italic, code, bdo, div, img, table}
 }
 
+func extractImageSource(src string, ok bool, selec *goquery.Selection) string {
```

Comment:
It seems we don't need **ok** as input parameter here

Reply:
Fixed